packages/create-react-app/index.js

```diff
@@ -105,7 +105,7 @@ function createApp(name, verbose, version) {
   checkAppName(appName);
 
   if (!pathExists.sync(name)) {
-    fs.mkdirSync(root);
```

Comment:
Ah sorry, I thought we were already using `fs-extra`. Turns out we do in `react-scripts`, but not not in `create-react-app`.